user/application_user.md

```diff
@@ -30,11 +30,15 @@ It is important this container be built for ARM architecture, so that it can run
 
 If your service requires any types of peripherals, you can select these from a list when creating your service. Your service will only be able to run on devices with this peripheral type configured.
 
+If your service wants single-tenancy over any type of resource, select the type of resource. This metadata allows us to avoid scheduling applications at the same time that might be affected by the presence of others. For example, if you are collecting network bandwidth information, you will likely not want others to run bandwidth tests simultaneously, but it may be OK to run this at the same time another application reads from a sensor. If your application generally requires single-tenancy, you can enable this when submitting an "application."
```

Comment:
If this isn't defined elsewhere, can we add a sentence to clarify what we mean by resource?

Reply:
I'm removing this section, as I think it'll be unused for now, and is confusing.